index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Pete Heyes - Developer Chap</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css"/>
+    <link rel="stylesheet" href="css/normalize.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Yellowtail" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+    <div class="split1">
+        <nav>
+            <ul class="nav-items">
+                <li><a href="#home" class="link">Home</a></li>
+                <li><a href="#portfolio" class="link">Portfolio</a></li>
+                <li><a href="#about" class="link">About Me</a></li>
+                <li><a href="#contact" class="link">Contact</a></li>
+            </ul>
+                <img class="ham" src="img/bars.svg"/>
+            <div class="icons-container">
+                <a href="https://github.com/HazDuck" target="_blank"><img src="img/github.png"/></a>
+                <a href="https://twitter.com/peteheyes" target="_blank"><img src="img/twitter.png"/></a>
+                <a href="https://uk.linkedin.com/in/pete-heyes-b7b98985" target="_blank"><img src="img/in.png"/></a>
+            </div>
+        </nav>
+        <main id="home" class="home">
+            <p class="headline gutter">Hi I'm Pete...</p>
+            <div class="portrait"></div>
+            <p class="quote gutter">Full Stack Web Developer (TBC) based in Bath</p>
+        </main>
+    </div>
+    <main id="portfolio" class="portfolio">
+        <h4 class="brand">My Projects</h4>
+        <div class="port-container">
```

Comment:
not sure you need this div? could move the styles on it elsewhere?

Reply:
cant work out how to do this so will make changes when i make improvements due to time constraints